CHANGELOG.adoc

```diff
@@ -70,6 +70,7 @@ The result returned and captured in this variable name will not be the real node
 - https://github.com/eclipse-sirius/sirius-components/issues/1527[#1527] [diagram] Add the new layout data structure which will be used by the new version of our layout algorithm.
 This new data structure will be used by the frontend if the name of a diagram ends with `__EXPERIMENTAL`.
 This will allow us to easily compare the current layout algorithm and the new one.
+- https://github.com/eclipse-sirius/sirius-components/issues/1883[#1883] [sirius-web] Add some metadata to the projects
```

Comment:
I know the actual issue does not provide any more details, but in the changelog we should be more precise about what was added.